README.md

```diff
@@ -12,9 +12,11 @@ The script `docker/build_docker.sh` is used to build all modified docker images.
 
 ## Getting Started
 **Pre-requisites:**
```

Comment:
shouldn't homebrew be the first prerequisite ?

Reply:
I think it's ok. Don't want to make this mac specific too much.